### ISSUE TITLE
support top of file # encoding lines

### ISSUE THREAD

**Issue by NorthIsUp** [timothycrosley/isort#260]
While not specifically import sorting this is also a top-of-file issue.

https://www.python.org/dev/peps/pep-0263/

thoughts?

**Comment by luzfcb** [comment]
@timothycrosley 

I could reproduce the error with the following code snippet

foobar.py file
```
# -*- coding: utf-8 -*-
from django.db import models
#import json as simplejson
from myproject.models import Servidor
import reversion


import logging
```

after execute ```isort foobar.py```, results:

```

import logging

import reversion
from django.db import models

# -*- coding: utf-8 -*-
#import json as simplejson
from myproject.models import Servidor
```

**Comment by timothycrosley** [comment]
I have verified this is an issue and Ill be working hard to fix it before the next release 

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
Fixed in version 4.1.0

Thanks!

~Timothy

**Comment by luzfcb** [comment]
Thanks
